testing/unit_tests/test_tensorboard_save.py

```
@@ -19,13 +19,13 @@ def test_tensorboard_save():
     inp = torch.tensor(np.zeros((1, 1, 15, 15)))
     gt = torch.tensor(np.zeros((1, 1, 15, 15)))
     pred = torch.tensor(np.zeros((1, 1, 15, 15)))
-    dpath = os.path.join(__tmp_dir__, "test_tensorboard_save")
-    os.makedirs(dpath)
-    writer = SummaryWriter(log_dir=dpath)
+    dpath = Path(__tmp_dir__, "test_tensorboard_save")
+    dpath.mkdir(parents=True)
```

I recommend adding optional ignore error argument.